### PR TITLE
Couple of changes to cmd.js

### PR DESCRIPTION
**Pull request by zcecc22**
Commits documented and signed off. Ready to discuss as needed.

### CODE REVIEW

**Comment by feross** [comment]
I'm curious why you need the `--exit` option? The auto-closing behavior seems to work fine for me. What's your use case?

File: bin/cmd.js [hunk 1]
```diff
@@ -538,58 +548,59 @@ function drawTorrent (torrent) {
...
-        '{green:Blocked peers:} {bold:' + torrent.numBlockedPeers + '}  ' +
-        '{green:Hotswaps:} {bold:' + hotswaps + '}'
-      )
-    }
+    line(
+    '{green:Queued peers:} {bold:' + torrent.swarm.numQueued + '}  ' +
+    '{green:Blocked peers:} {bold:' + torrent.numBlockedPeers + '}  ' +
+    '{green:Hotswaps:} {bold:' + hotswaps + '}'
+    )
+
```
Comment (feross) [inline bin/cmd.js]: This is pretty advanced information. Why do you want to show it to all users?

**Comment by zcecc22** [comment]
Seeding the torrent when done downloading.

So far the only work around Ive found was to relaunch with seed instead of
download which is not efficient on several gb torrents (long rehash).

Is'nt it best practice to keep sharing by default?

BR,
V

**Comment by feross** [comment]
Yeah, you make a good point. My concern is that lots of folks use the command line to fetch files and then do other things with them. Example:

```bash
webtorrent magnet_uri --stdout | some-other-program
```

We can make it so that `--stdout` implies `--exit`.

In the future, please send separate PRs for each change you want to make so they can be discussed and merged individually.

**Comment by feross** [comment]
I like this PR, except for the changes you made to the UI. Can you remove those? It doesn't make sense to me to hide the detailed peer info under `--verbose` when we're already clearing the entire terminal and taking up the whole terminal buffer.

In another PR/issue, I'm open to making a more minimalist download UI similar to the npm3 one:



And we can hide the current one behind `--verbose`.

Sound good?

**Comment by zcecc22** [comment]
Sounds good.

I ll open new PRs for the port and exit including the implicit for stdout.

Feel free to drop this one.

Cheers,
V

**Comment by feross** [comment]
Thanks for opening new PRs. That's really awesome of you. I'll look for those.

**Pull request closed without merging**